src/app/zap-templates/zcl/data-model/chip/basic-information-cluster.xml

```diff
@@ -15,34 +15,34 @@ See the License for the specific language governing permissions and
-    <attribute side="server" code="5"  define="NODE_LABEL"                type="char_string"               length="32"  default=""  writable="true"                >NodeLabel</attribute>
-    <attribute side="server" code="6"  define="LOCATION"                  type="char_string"               length="2"   default=""  writable="true"                >Location</attribute>
-    <attribute side="server" code="7"  define="HARDWARE_VERSION"          type="int16u"                                 default="0"                                >HardwareVersion</attribute>
-    <attribute side="server" code="8"  define="HARDWARE_VERSION_STRING"   type="char_string" minLength="1" length="64"                                             >HardwareVersionString</attribute>
-    <attribute side="server" code="9"  define="SOFTWARE_VERSION"          type="int32u"                                 default="0"                                >SoftwareVersion</attribute>
-    <attribute side="server" code="10" define="SOFTWARE_VERSION_STRING"   type="char_string" minLength="1" length="64"                                             >SoftwareVersionString</attribute>
-    <attribute side="server" code="11" define="MANUFACTURING_DATE"        type="char_string" minLength="8" length="16"                              optional="true">ManufacturingDate</attribute>
-    <attribute side="server" code="12" define="PART_NUMBER"               type="char_string"               length="32"                              optional="true">PartNumber</attribute>
-    <attribute side="server" code="13" define="PRODUCT_URL"               type="long_char_string"          length="256"                             optional="true">ProductURL</attribute>
-    <attribute side="server" code="14" define="PRODUCT_LABEL"             type="char_string"               length="64"                              optional="true">ProductLabel</attribute>
-    <attribute side="server" code="15" define="SERIAL_NUMBER"             type="char_string"               length="32"                              optional="true">SerialNumber</attribute>
-    <attribute side="server" code="16" define="LOCAL_CONFIG_DISABLED"     type="boolean"                                default="0" writable="true" optional="true">LocalConfigDisabled</attribute>
-    <attribute side="server" code="17" define="REACHABLE"                 type="boolean"                                default="1"                 optional="true">Reachable</attribute>
-    <attribute side="server" code="18" define="UNIQUE_ID"                 type="char_string"               length="32"                              optional="true">UniqueID</attribute>
+    <attribute side="server" code="0" define="INTERACTION_MODEL_VERSION" type="int16u" default="1">InteractionModelVersion</attribute>
+    <attribute side="server" code="1" define="VENDOR_NAME" type="char_string" length="32">VendorName</attribute>
```

Comment:
this is just making thing hard to read imo , i prefer straight column

Reply:
We need a formatting tool for XML. My XML editor is normalizing the space.

Reply:
I'll take this rename out of the commit. Cam wants both of the Basic clusters renamed to Basic Information. That can be done in a separate PR so that we stop getting so many conflicts.